clients/java-logger/library/src/test/java/com/abcxyz/lumberjack/auditlogclient/processor/JustificationProcessorTests.java

```diff
@@ -102,15 +106,25 @@ public void processShouldThrowExceptionWhenFailingToValidateToken() throws Excep
     doThrow(new JwkException("")).when(jvsClient).validateJWT(token);
 
     JustificationProcessor processor = new JustificationProcessor(jvsClient);
-    assertThrows(LogProcessingException.class, () -> processor.process(token, auditLogRequest));
+    assertThrows(LogProcessingException.class, () -> processor.process(getAuditLogRequest(token)));
   }
 
   @Test
   public void processShouldThrowExceptionWithoutLogReqPayload() throws Exception {
     JustificationProcessor processor = new JustificationProcessor(jvsClient);
     assertThrows(
         IllegalArgumentException.class,
-        () -> processor.process("token", AuditLogRequest.newBuilder().build()));
+        () -> processor.process(AuditLogRequest.newBuilder().build()));
+  }
+
+  @Test
+  public void processLogReqWithNullJVSToken() throws Exception {
+    verifyNoOp(auditLogRequest);
+  }
+
+  @Test
+  public void processLogReqWithEmptyJVSToken() throws Exception {
+    verifyNoOp(getAuditLogRequest(""));
```

Comment:
Could we name the helper to something like `getAuditLogRequestWithJvsToken("")` to make it obvious that the empty string we are passing is the JVS token?

Reply:
Yes, we could, will change.